src/test/system_tests/ceph_s3_tests/s3-tests-lists/s3_tests_pending_list.txt

```diff
@@ -136,4 +136,5 @@ s3tests_boto3/functional/test_s3.py::test_cors_presigned_put_object
 s3tests_boto3/functional/test_s3.py::test_cors_presigned_put_object_with_acl
 s3tests_boto3/functional/test_s3.py::test_cors_presigned_put_object_tenant
 s3tests_boto3/functional/test_s3.py::test_cors_presigned_put_object_tenant_with_acl
-s3tests_boto3/functional/test_s3.py::test_object_presigned_put_object_with_acl_tenant
\ No newline at end of file
+s3tests_boto3/functional/test_s3.py::test_object_presigned_put_object_with_acl_tenant
```

Comment:
can you please remove this change as well

Reply:
sure